common/test/acceptance/tests/video/test_video_events.py

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
For what it's worth, we do have video events tests in our JS Tests suite. (Though, I can't guarantee for you that they are viable...just an FYI.)